_includes/footer/desktop.html

```diff
@@ -0,0 +1,56 @@
+<footer class="footer-desktop" role="contentinfo">
```

Comment:
There's different functionality and content between the two. While we could do this with CSS, it's easier to hide and show the two at different screen sizes and has no negative impact on accessibility. I'd mark this as an enhancement, the advantage of having one markup is ease of updating but these should not be having a lot of changes to start. We can always enhance from here.